src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointEvidence.java

```diff
@@ -132,30 +157,66 @@ private static TemplateEnd findTemplateEnd( final GATKRead read ) {
         private static final String SA_TAG_NAME = "SA";
         private final String cigar;
         private final String tagSA;
+        private List<SVInterval> supplementaryAlignments;
 
         public SplitRead( final GATKRead read, final ReadMetadata metadata, final boolean atStart ) {
             super(read, metadata, atStart ? read.getStart() : read.getEnd(), UNCERTAINTY);
             cigar = read.getCigar().toString();
             if ( cigar == null ) throw new GATKException("Read has no cigar string.");
             tagSA = read.getAttributeAsString(SA_TAG_NAME);
```

Comment:
I think that kryo's `writeString` and `readString` methods both handle null so I'm not sure I see an asymmetry here.